### ISSUE TITLE
Failed to parse file : TypeError [ERR_INVALID_ARG_TYPE]

### ISSUE THREAD

**Issue by projectzero** [emmercm/igir#249]
### Paste the command

igir copy --dat "E:\ROMVault_V3.4.5\DatRoot\7 Retool 1G1R [Redump Console]\Sega - Saturn [EN] (2023-01-27 03-45-33) (Retool 2023-01-30 10-07-00) (324) (-bBdPrv).dat" --input "Z:\3 Redump [Console]\Sega - Saturn" --output "Z:\7 Retool 1G1R [Redump Console]\Sega - Saturn [EN] (Retool)"

### Describe the bug

Attempting to process a 1G1R dat for Sega Saturn and I receive the attached error. This particular game isn't in the DAT file, just the directory.

![IGIR error](https://user-images.githubusercontent.com/3341925/215564158-80ebab79-9b47-4bae-9e05-cc8484671f85.PNG)

### Expected behavior

No error based on file name. 

### Debug logs

![IGIR error 2](https://user-images.githubusercontent.com/3341925/215615374-0eca3185-1f81-4259-9c23-ceab62fd7ae2.PNG)

### igir version

1.0.2

### Node.js version

18.13.0

### Operating system

Windows 10

**Comment by emmercm** [comment]
First, I didn't realize half of the CLI icons don't show in Windows, so that's definitely something I'll fix.


But in your specific case, is it just that one file in your set that's problematic? It looks like `igir` is able to skip the file at least gracefully, but the screenshot was grabbed before any writing happened.

I have a feeling there's something unique with that 7z file. Could you list the contents of it? Using the standalone [7zip `7zr.exe`](https://www.7-zip.org/download.html) the command is:

```shell
7zr l -slt -ba test.7z
```

**Comment by projectzero** [comment]
Here's what came up using the command that worked for me (7zr didn't but 7z did, I'm guessing it's the same thing)

![IGIR error 4](https://user-images.githubusercontent.com/3341925/215635917-4a33db70-0bff-412d-9dee-e23b1d821860.PNG)

**Comment by emmercm** [comment]
Oh, I wonder if it's because there is an equals sign in the compressed filenames. I'll have to do some testing with this, it might be a bug in a library. Thank you!

**Comment by emmercm** [comment]
Yep, that's exactly what it is, I opened an issue with the library: https://github.com/onikienko/7zip-min/issues/70.

**Comment by projectzero** [comment]
Thanks for looking into it so quickly!

**Comment by emmercm** [comment]
This will be fixed by https://github.com/emmercm/igir/pull/258 in the next release!
